README.md

```diff
@@ -6,19 +6,19 @@
 
 The missing DOM integration testing tool for [Next.js][next-github].
 
-Given a Next.js route, this library will return an instance of the matching page component instantiated with the **properties** derived by Next.js' [**routing system**][next-docs-routing] and [**data fetching methods**][next-docs-data-fetching].
+Given a Next.js route, this library will **mount the matching page in JSDOM**, provided with the expected **props** derived from Next.js' [**routing system**][next-docs-routing] and [**data fetching methods**][next-docs-data-fetching].
 
 ```js
-import { render, screen, fireEvent } from '@testing-library/react';
```

Comment:
Would it make sense to make this a section with "Using @testing-library/react" header and provide another example how library could be used without it?

Reply:
What if we did the opposite? :) Like adding a section where we explain that `next-page-tester` does nothing more then appending stuff in in the (mocked) DOM and call React.hydrate on it.

I assume `@testing-library` is almost a de-facto standard library and I'd like to not clutter the initial example with too 
many DOM API's (considering we simulate a click event). How do you see it?

Reply:
Perhaps we should create an `examples` folder where we can "gather" some patterns to not clutter `README.md`. It can be an ongoing effort as a "response" to potential issues users might face.